README.md

```diff
@@ -3,6 +3,55 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+`tap-fixerio` works to pull currency exchange rates from the Fixer API
+
+### Install and Run
+
+First, make sure Python 3 is installed on your system or follow these
+installation instructions for [Mac] or
+[Ubuntu].
+
+First, get an access key from [fixer.io](http://fixer.io). You need at least
+the basic plan to get exchange rates from a base currency (such as USD).
+
+Convert config.sample.json to config.json; fill out your parameters.
+
+It's recommended to use a virtualenv:
+
+```bash
+ python3 -m venv ~/.virtualenvs/tap-fixerio
+ source ~/.virtualenvs/tap-fixerio/bin/activate
+ pip install -U pip setuptools
+ pip install -e '.[dev]'
+```
+
+
+These commands will install `tap-fixerio`  with pip,
+and then run it:
+
+```bash
+> pip install target-csv
+> tap-fixerio
+  INFO Replicating the latest exchange rate data from fixer.io
+  INFO Tap exiting normally
+```
+
+The data will be written to a file called `exchange_rate.csv` in your
+working directory.
+
+```bash
+› cat exchange_rate.csv
```

Comment:
This `> ` is useful (although I'd make it `$` as that's the default for bash `PS1` prompts.) as it shows that it's a console log. I would remove the `bash` language tag in this instance since it's just a plaintext log.

Reply:
This was copied from here: https://github.com/singer-io/target-csv

I don't have strong feelings on it, but I thought being consistent was important?

Reply:
`$` is the correct value so it'd actually be better to open a PR to fix `target-csv` than it is to use `>` here.